hls-test-utils/hls-test-utils.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version: 2.4
 name:          hls-test-utils
-version:       1.0.0.1
+version:       1.0.1.0
```

Comment:
For the version `a.b.c.d`, in which case should we bump `c` instead of `d`?

Reply:
For this package, I bump `c` as this exposes a new binding, following the PVP:
> *Non-breaking change.* Otherwise, if only new bindings, types, classes, non-orphan instances or modules (but see below) were added to the interface, then *A.B* **MAY** remain the same but the new *C* **MUST** be greater than the old *C*. Note that modifying imports or depending on a newer version of another package may cause extra non-orphan instances to be exported and thus force a minor version change.

For other cases, I believe I bump `c` part only for packages that expose different behaviors. If there is any mistake, please let me know.